imgparse/imgparse.py

```diff
@@ -803,6 +831,18 @@ def get_unique_id(image_path, xmp_data=None):
     :param xmp_data: used internally for memoization. Not necessary to supply.
     :return: unique image id in the form <session_id>_<image_id>
     """
+    # DJI CaptureUUID
+    make, model = get_make_and_model(image_path)
+    if make == "DJI":
+        try:
+            return str(xmp_data["drone-dji:CaptureUUID"])
```

Comment:
DJI stores CaptureUUID in a different set of tags and uses a UUID that is stored across bands. 

![image](https://github.com/user-attachments/assets/d77e96a5-17c5-42fd-ba71-089d7163161d)

Reply:
Please replace with `xmp_tags = xmp.get_tags(make)` and update the DJI xmp tags enum to keep this consistent with all other xmp parsing